lib/collections/transform/cartOrder.js

```diff
@@ -203,14 +203,16 @@ export const cartOrderTransform = {
     const subtotals = this.getSubtotalByShop();
     const taxes = this.getTaxesByShop();
     const shipping = parseFloat(this.getShippingTotal());
+    const discount = parseFloat(this.getDiscounts());
```

Comment:
I know I said we should be reflecting discount everywhere but this is actually the exception. We haven't finalized how we are handling discounts in a multi-shop scenario and I think this is going to give us odd results, so let's back out this change for now.

Reply:
@zenweasel when I add a discount, it's not reflected in `Total` displayed on Orders page.